### DISCUSSION TITLE
How to properly display a Home-Assistant "Timer"

### DISCUSSION THREAD

**Discussion by iamthe1st** [Blackymas/NSPanel_HA_Blueprint#1760] [Q&A]
Hi,
i have two laundry timers setup in Home Assistant, which i can start by pressing the right button of the NSPanel which then does trigger an Automation which starts the pre-configured Timer.
On the Web-Interface i can see, that the timer has started and the 1hour 33minutes counts down.

I setup the Blueprint, so that the Timer-Entity is displayed in Sensors-Page, but it only shows me "idle" and switches to "active" if the timer has started.

How is it possible to display the remaing time instead? 
A simple countdown of the minutes would be fine.

![image](https://github.com/Blackymas/NSPanel_HA_Blueprint/assets/11601070/b72b9555-1fcc-4735-a580-16175f7e4bb1)

**Comment by edwardtfn** [comment]
We don't have a way to visualize that yet, so I've added as an enhancement request:

- #1772 

Let's continue the discussions on that one. ;)
